viral/response_profiles.py

```
@@ -0,0 +1,179 @@
+        hue_order=["WT", "NLGF"],
+        dodge=True,
+        linewidth=1,
+        edgecolor="black",
+    )
+    ttest_ind(
+        df[(df["genotype"] == "WT") & (df["stage"] == "Trained")]["variance_explained"],
+        df[(df["genotype"] == "NLGF") & (df["stage"] == "Trained")][
+            "variance_explained"
+        ],
+    )
+    1 / 0
```

Ruff (0.14.14)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)